apps/qaqc/src/checkbook/checkbook.py

```diff
@@ -0,0 +1,108 @@
+import streamlit as st
+import pandas as pd
+from helpers import get_data
+import folium
+from streamlit_folium import st_folium
+import geopandas 
+
+
+def get_category_data(df):
+    category_data = df.groupby("final_category", as_index=False).sum()
+    return category_data
+
+
+def get_geometry_data(df):
+    geometry_data = df.groupby("has_geometry", as_index=False).sum()
+    return geometry_data
+
+def output_map(df):
+    df_clean = df.dropna(subset=['geometry'])
+    geometries = geopandas.GeoSeries.from_wkt(df_clean['geometry'])
+    gdf = geopandas.GeoDataFrame(df_clean, geometry=geometries, crs="EPSG:4326")
```

Comment:
Unsure about this, so wondering if you tested - does specifying `geometry="geometry"` without defining the geoseries above work? Or do you specifically need the `GeoSeries.from_wkt` to read it properly

Reply:
I believe just using the "geometry" column without using GeoSeries.from_wkt was giving me an error